test/dlc_tests/sanity/test_security_check.py

```diff
@@ -10,9 +11,14 @@
 from invoke import run
 from invoke import Context
 
-from test.test_utils import get_account_id_from_image_uri, get_region_from_image_uri, login_to_ecr_registry
+from test.test_utils import (
+    LOGGER, get_account_id_from_image_uri, get_dockerfile_path_for_image
+)
 from test.test_utils import ecr as ecr_utils
-from test.test_utils.ecr import CVESeverity
+from test.test_utils.security import CVESeverity, ScanVulnerabilityList
+
+
+MINIMUM_SEV_THRESHOLD = "MEDIUM"
```

Comment:
This will be set back to "HIGH" until we actually have allowlists for dockerfiles.